write.go

```diff
@@ -49,6 +49,8 @@ func isAscii(s string) bool {
 type Writer struct {
 	io.Writer
 
+	AllowUnsyncLiterals bool
```

Comment:
"async" is generally used instead of "unsync"

Reply:
ok